### ISSUE TITLE
Implement Elevation limits

### ISSUE THREAD

**Issue by kmharrington** [simonsobs/scheduler#107]
The SATs have minimum elevation limits. I just noticed those aren't in the configurations / never checked here. So far hasn't been an issue because we just know them (48 deg for satp1 and satp3, satp2 is set there right now but might get lowered). But we should have the software check for these.

**Comment by guanyilun** [comment]
is this as simple as adding assert el > 48 or similar?

**Comment by kmharrington** [comment]
It does need to be configurable at some level, since I think SATp2's will end up being different. I guess the question is if it's better for it to be a "rule" or something that causes and error.

**Comment by kmharrington** [comment]
yeah sounds good. We'll want to check all of the blocks not just the calibration target ones. Would also do `el >= el_min` instead of just equals.
